.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,20 +1,56 @@
 ---
-name: Feature request
+name: 💡 Feature request
 about: Suggest an idea for this project
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!--
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+Welcome! - We kindly ask you to:
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+  1. Fill out the issue template below 
+  2. Use Gitter if you have a question rather than a bug or feature request.
+    Gitter community channel is at: https://gitter.im/devfile/community
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+Thanks for understanding, and for contributing to the project!
```

Comment:
See my earlier comment. If you do make this my suggested, optional change, you'll want to make the same change here :)

Reply:
updated